internal/testutil/runmain.go

```diff
@@ -5,6 +5,8 @@
 package testutil
 
 import (
+	realContext "context"
```

Comment:
We should rename the `testutil.context()` function and not rename this.

Reply:
done